src/solvers/flattening/boolbv.h

```diff
@@ -234,19 +233,6 @@ class boolbvt:public arrayst
   // unbounded arrays
   bool is_unbounded_array(const typet &type) const override;
 
-  // quantifier instantiations
-  class quantifiert
-  {
-  public:
-    exprt expr;
-    literalt l;
-  };
-
-  typedef std::list<quantifiert> quantifier_listt;
-  quantifier_listt quantifier_list;
-
-  void post_process_quantifiers();
```

Comment:
What's the rationale?  If we are going to do anything smart with quantifiers, this would be the place / hook to do it.

Reply:
I'm hard pushed to think of an advanced technique for handling quantifiers that won't use this or something like this.